src/components/Auth/AuthHookForm.tsx

```diff
@@ -0,0 +1,40 @@
+"use client";
+import { LinkbraryIcon } from "@/src/asset";
+import Link from "next/link";
+import { useForm } from "react-hook-form";
+import InputField from "./InputField/InputField";
+import styles from "./AuthHookForm.module.scss";
+
+export default function AuthHookForm() {
+  const { handleSubmit } = useForm();
+  // register : value 관련
+  // watch : input 상태감지
+  // formState : submit error 감지
+
+  return (
+    <main className={styles.authForm}>
+      <header className={styles.header}>
+        <Link href={"/"}>
```

Comment:
문자열은 중괄호 없이 그대로 표현해주시면 더 좋을 것 같습니다~
```suggestion
        <Link href="/">
```

Reply:
routing 경로는 상수화를 해서 다뤄주면 더 좋습니다!

상수를 만들어두고,
```ts
export const ROUTE_PATHS = {
  HOME: '/',
};
```

사용하는 곳에서는
```suggestion
        <Link href={ROUTE_PATHS.HOME}>
```